apps/system/js/bluetooth_transfer.js

```diff
@@ -62,19 +68,19 @@ var BluetoothTransfer = {
       this.sendFileViaHandover.bind(this));
 
     Service.registerState('isSendFileQueueEmpty', this);
+    Service.registerState('isFileTransferInProgress', this);
   },
 
   getDeviceName: function bt_getDeviceName(address) {
-    return new Promise(function(resolve) {
+    return new Promise((resolve) => {
       var _ = navigator.mozL10n.get;
-      var adapter = Bluetooth.getAdapter();
-      if (adapter == null) {
+      var _adapter = Service.query('Bluetooth.getAdapter');
```

Comment:
Since the `_adapter` is a locale variable in this function, you don't need to prefix with `_`. Just use `adapter` is fine.